lab-anthony/model/drink.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+const mongoose = require('mongoose');
+
+const drinkSchema = mongoose.Schema({
+  drinkName: {type: String, required: true},
+  drinkDesc: {type: String, required: true},
+  locId: {type: mongoose.Schema.ObjectId, required: true}
```

Comment:
You might want to call this cafeId or something to keep things as clear as possible. Where is the ref here?